playbooks/openshift-logging/private/config.yml

```diff
@@ -70,11 +80,12 @@
         groups: oo_elasticsearch_nodes
         ansible_ssh_user: "{{ g_ssh_user | default(omit) }}"
         ansible_become: "{{ g_sudo | default(omit) }}"
-      with_items: "{{ openshift_logging_elasticsearch_hosts }}"
+      with_items: "{{ groups['OSEv3'] }}"
       changed_when: no
       run_once: true
       delegate_to: localhost
       connection: local
+      when: hostvars[item]['openshift']['common']['ip'] in openshift_logging_elasticsearch_hosts
```

Comment:
This is a clever solution.  I would just say, we should add some comments to this file about what we're doing here because it might not be obvious later.

Reply:
sure thing

Reply:
added some comments to explain what we're doing with `add_host` and why we're collecting openshift_facts above as well